test/microsoft/dynamics/Delete/Delete.test.js

```
@@ -0,0 +1,61 @@
+
+        sinon.reset();
+    });
+
+
+    beforeEach(function() {
+
+        context = testUtils.createMockContext();
+        // Set properties to generate output port options.
+        context.messages = { in: { content: { id: 'testId' } } };
+        context.properties = {};
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)